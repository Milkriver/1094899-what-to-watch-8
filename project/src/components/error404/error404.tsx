export function Error404(): JSX.Element {
  return (
    <div className="error404">
      <div>404 Not Found</div>
      <a href="main.html" className="main">Вернуться на главную страницу</a>
    </div>
  );
}
