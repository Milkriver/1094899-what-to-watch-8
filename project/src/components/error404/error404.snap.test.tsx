import {render} from '@testing-library/react';
import { createMemoryHistory } from 'history';
import { Router } from 'react-router';
import { Error404 } from './error404';


describe('Component: Error404', () => {
  const history = createMemoryHistory();
  const page = (
    <Router history={history}>
      <Error404 />;
    </Router>
  );
  test('should render correctly', () => {
    const {container} = render(page);
    expect(container).toMatchSnapshot();
  });
});

