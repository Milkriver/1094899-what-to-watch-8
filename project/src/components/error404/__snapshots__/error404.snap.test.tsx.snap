// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Error404 should render correctly 1`] = `
<div>
  <div
    class="error404"
    style="display: flex; justify-content: center; height: 100vh; align-items: center;"
  >
    <div
      style="margin-right: 10px;"
    >
      404 Not Found 
    </div>
    <a
      class="main"
      href="/"
    >
      Вернуться на главную страницу
    </a>
  </div>
  ;
</div>
`;
